build/localnet/Tiltfile

```diff
@@ -120,8 +120,11 @@ docker_build_with_restart(
     "client-image",
     root_dir,
     dockerfile_contents="""FROM debian:bullseye
-RUN apt-get update && apt-get install -y procps
-WORKDIR /
+RUN apt-get update && apt-get install -y procps bash-completion
```

Comment:
Could we also are jq here? I find the output of RPC easier to grok when I can actually read it 😂

Reply:
i like that actually. I did find myself installing it lol